main.go

```diff
@@ -1,48 +1,106 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"github.com/Azure/batch-insights/pkg"
+	log "github.com/sirupsen/logrus"
 	"os"
 	"strings"
 )
 
+func parseListArgs(value string) []string {
+	names := strings.Split(value, ",")
+	for i := range names {
+		names[i] = strings.TrimSpace(names[i])
+	}
+	return names
+}
+
+func getenv(key string) *string {
+	value := os.Getenv(key)
+	if len(value) == 0 {
+		return nil
+	}
+	return &value
+}
+
+func initLogger() {
+	log.SetFormatter(&log.TextFormatter{
+		FullTimestamp:    false,
+		DisableTimestamp: true,
+		ForceColors:      true,
+	})
+}
+
 func main() {
-	var appInsightsKey = os.Getenv("APP_INSIGHTS_INSTRUMENTATION_KEY")
-	var poolId = os.Getenv("AZ_BATCH_POOL_ID")
-	var nodeId = os.Getenv("AZ_BATCH_NODE_ID")
-	var processNamesStr = os.Getenv("AZ_BATCH_MONITOR_PROCESSES")
+	initLogger()
+	disableArg := flag.String("disable", "", "List of metrics to disable")
```

Comment:
Not sure if we own this, but it isn't really a flag, more an argument.

Reply:
You mean the flag import?
That's the standard go argument parser